libraries/rtos/rtx/TARGET_CORTEX_A/RTX_CM_lib.h

```diff
@@ -85,16 +85,24 @@ OS_RESULT _os_mut_wait    (uint32_t p, OS_ID mutex, uint16_t timeout) __svc_indi
 
 #if (OS_TIMERS != 0)
 #define OS_TASK_CNT (OS_TASKCNT + 1)
+#ifndef __MBED_CMSIS_RTOS_CA9
```

Comment:
We discussed the readability.
As you know, the code of RTOS for the Cortex-A9 has been diverted from the DS-5.
So in view of the maintenance, we would like to leave "ifdef" because we want to clarify what has been changed from DS-5.
However, we will modify the following two points.
　·If ifdef and ifndef are mixed, unify it to ifdef
　·Replace Macro "__MBED_CMSIS_RTOS_CA9" with another macro.

How about our proposal ?